orchestrator/api/tools.py

```
@@ -31,6 +31,41 @@
+            logger.info(
+                "[PRD-183 S2] Auto-sync complete for workspace %s", workspace_id,
+            )
+        except Exception as e:  # noqa: BLE001 — background task must not raise
+            logger.warning(
+                "[PRD-183 S2] Auto-sync failed for workspace %s: %s",
+                workspace_id, e,
+            )
+        finally:
+            db.close()
+
+    _asyncio.create_task(_run())
```

Ruff (0.15.20)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)